src/main/java/com/twilio/sdk/TwilioTaskRouterClient.java

```diff
@@ -42,6 +42,10 @@
 	public TwilioTaskRouterClient(final String accountSid, final String authToken) {
 		super(accountSid, authToken, "https://taskrouter.twilio.com");
 	}
+	
+	public TwilioTaskRouterClient(final String accountSid, final String authToken, final String endpoint) {
+        super(accountSid, authToken, endpoint);
+    }
```

Comment:
indent looks off here? this codebase is tab-indented if you're configured for spaces